packages/kolibri-common/components/Cards/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+
+  
+  <KCard
+    :to="to"
+    :headingLevel="headingLevel"
+    layout="horizontal"
+    thumbnailDisplay="large"
+    :title="contentNode.title"
+    :thumbnailSrc="thumbnailSrc"
+    thumbnailScaleType="centerInside"
+    thumbnailAlign="right"
+    :style="{ height: '172px', margin: '16px 0 16px 0' }"
```

Comment:
Non-blocking question: 

Would any of these other `KCard` props be good to accept as props in `AccessibleFolderCard`? Like - would we ever want to use this card in a context where we want a different `thumbnail(Align|Display|ScaleType)` or `layout` values?

Reply:
Sorry I missed this comment but I get your point, according to @MisRob suggestion here https://github.com/learningequality/kolibri/pull/12418#discussion_r1733970176, states otherwise.

Reply:
@nucleogenesis I am certain that scale type will always be the default `centerInside` in all card components because their thumbnails are user-uploaded and don't have consistent ratios (you can see more in [KImg -> Usage > Scaling](https://design-system.learningequality.org/kimg/#usage)).

Regarding the layout related settings, each of the three cards has its layout configuration. As far as we know from the designs, we won't be using multiple configurations for the same card type, however I can imagine this could easily change depending on the future designs.

So if it feels important to think through it more carefully, perhaps each prop deserves its own consideration.

In the comment you reference @AllanOXDi, I don't mean to suggest we never use these extra props, but rather that we start using them when there's an actual need. You could argue otherwise depending on a mental model for these cards. I emphasized simplicity at this point, but nothing blocking. You can take it as one of the perspectives on designing interfaces in general and welcome to choose which way to go.

Reply:
@nucleogenesis I am certain that scale type will always be the default `centerInside` in all card components because their thumbnails are user-uploaded and don't have consistent ratios (you can see more in [KImg -> Usage > Scaling](https://design-system.learningequality.org/kimg/#usage)).

Regarding the layout related settings, each of the three cards has its layout configuration. As far as we know from the designs, we won't be using multiple configurations for the folder card I commented on, however I can imagine this could easily change depending on the future designs and card type (there's more layouts for the resource card, I believe).

So if it feels important to think through it more carefully, perhaps each prop deserves its own consideration.

In the comment you reference @AllanOXDi, I don't mean to suggest we never use these extra props, but rather that we start using them when there's an actual need. You could argue otherwise depending on a mental model for these cards. I emphasized simplicity at this point, but nothing blocking. You can take it as one of the perspectives on designing interfaces in general and welcome to choose which way to go.

Reply:
But also at this point we won't  allowing  a lot customization of these cards beyond the design. I do remember  the reason as to why  we allowed  customization further using `thumbnail(Align|Display)` was handle the edge cases that these cards provide.

Reply:
These make your point valid @MisRob. Keep it simple and wait until there is a need to

Reply:
> But also at this point we won't allowing a lot customization of these cards beyond the design. 

Oh yes, that's true, I just wanted to also acknowledge I see your point in having some customizations. Even though I didn't observe this for the folder card where the discussion originated, later I realized more broadly there's more to it, for example the resource card will likely need it in close future :) I double-checked designs earlier today to think more about this and noticed 

![Screenshot from 2024-09-04 13-42-56](https://github.com/user-attachments/assets/908cb022-6820-4279-933a-dd268ddbed65)

Anyways, good considerations here! You're welcome to put it together and arrive at something that makes most sense to you taking into account all these pieces.